core/src/main/java/fr/sncf/osrd/train/TrainStop.java

```diff
@@ -8,4 +11,19 @@ public TrainStop(double position, double stopDuration) {
         this.position = position;
         this.stopDuration = stopDuration;
     }
+
+    @Override
+    @ExcludeFromGeneratedCodeCoverage
```

Comment:
I've read about this trick and refrained from using it, I don't think it's legit

Reply:
I think I went a bit too fast with a `git add -p` and I had some weird stuff left over... It's not supposed to be here

Reply:
This annotation is working correctly.